app/views/layouts/application.html.haml

```diff
@@ -3,7 +3,7 @@
   %head
     %title Mycademy: Learn to Lead
     %link{:href => "https://fonts.googleapis.com/icon?family=Material+Icons", :rel => "stylesheet"}
-    %link{:href => "https://fonts.googleapis.com/css?family=Lato:400,300,700", :rel => "stylesheet", :type => "text/css"}/
+    %link{:href => "https://fonts.googleapis.com/css?family=Lato:400,300,700", :rel => "stylesheet", :type => "text/css"}
```

Comment:
Line is too long. [121/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace